src/StreamJsonRpc.Tests/JsonRpcClient20InteropTests.cs

```diff
@@ -301,7 +301,7 @@ public async Task ErrorResponseIncludesCallstack()
         };
         this.Send(errorObject);
         var ex = await Assert.ThrowsAsync<RemoteInvocationException>(() => requestTask);
-        var commonErrorData = ((JToken)ex.ErrorData).ToObject<CommonErrorData>();
+        var commonErrorData = ((JToken?)ex.ErrorData)!.ToObject<CommonErrorData>();
```

Comment:
💭 I would have taken the simpler route for this one

```suggestion
        var commonErrorData = ((JToken)ex.ErrorData!).ToObject<CommonErrorData>();
```